server/src/test/java/org/spine3/server/SubscriptionServiceShould.java

```diff
@@ -0,0 +1,291 @@
+
+import com.google.common.util.concurrent.MoreExecutors;
+import com.google.protobuf.Message;
+import io.grpc.stub.StreamObserver;
+import org.junit.Test;
+import org.spine3.base.Response;
+import org.spine3.client.Subscription;
+import org.spine3.client.SubscriptionUpdate;
+import org.spine3.client.Target;
+import org.spine3.client.Topic;
+import org.spine3.protobuf.AnyPacker;
+import org.spine3.server.stand.Stand;
+import org.spine3.server.storage.memory.InMemoryStandStorage;
+import org.spine3.server.storage.memory.InMemoryStorageFactory;
+import org.spine3.test.aggregate.Project;
+import org.spine3.test.aggregate.ProjectId;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertNotNull;
+import static org.junit.Assert.assertNull;
+import static org.junit.Assert.assertTrue;
+import static org.mockito.ArgumentMatchers.any;
+import static org.mockito.Mockito.never;
+import static org.mockito.Mockito.spy;
+import static org.mockito.Mockito.verify;
+
+/**
+ * @author Dmytro Dashenkov
+ */
+public class SubscriptionServiceShould {
+
+    /*
+     * Creation tests
+     * --------------
+     */
+
+    @Test
+    public void initialize_properly_with_one_bounded_context() {
+        final BoundedContext singleBoundedContext = newBoundedContext("Single");
+
+        final SubscriptionService subscriptionService = SubscriptionService.newBuilder()
+                                                                           .addBoundedContext(singleBoundedContext)
+                                                                           .build();
+
+        assertNotNull(subscriptionService);
+    }
+
+    @Test
+    public void initialize_properly_with_several_bounded_contexts() {
+        final BoundedContext firstBoundedContext = newBoundedContext("First");
+        final BoundedContext secondBoundedContext = newBoundedContext("Second");
+        final BoundedContext thirdBoundedContext = newBoundedContext("Third");
+
+
+        final SubscriptionService subscriptionService = SubscriptionService.newBuilder()
+                                                                           .addBoundedContext(firstBoundedContext)
+                                                                           .addBoundedContext(secondBoundedContext)
+                                                                           .addBoundedContext(thirdBoundedContext)
+                                                                           .build();
+
+        assertNotNull(subscriptionService);
+    }
+
+    @Test
+    public void be_able_to_remove_bounded_context_from_builder() {
+        final BoundedContext firstBoundedContext = newBoundedContext("Removed");
+        final BoundedContext secondBoundedContext = newBoundedContext("Also removed");
+        final BoundedContext thirdBoundedContext = newBoundedContext("The one to stay");
+
+
+        final SubscriptionService subscriptionService = SubscriptionService.newBuilder()
```

Comment:
We need to check that contexts are really removed from the builder.
Consider adding a method to the builder.